src/microbe_stage/MicrobeSpecies.cs

```diff
@@ -102,6 +102,12 @@ public void CalculateRotationSpeed()
         BaseRotationSpeed = MicrobeInternalCalculations.CalculateRotationSpeed(Organelles);
     }
 
+    public void UpdatePositionalProperties()
```

Comment:
I added RepositionToOrigin to the base species type as I think it'll be useful for many stages. Related to that maybe CalculateRotationSpeed should be put in the RepositionToOrigin call (on line 98)? Is there a reason why reposition to origin would be done without also re-calculating the rotation speed?

Reply:
Yeah, I don't think we'd ever want to call one without the other. It seems more likely for someone to do that on accident than on purpose.